Bottom: ffb69da953c142ae2b78c8f84dd6221188db1700
Top:    5a80a577ab16ea337ac8224226e1fda9436a0e94
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-05 16:31:26 +0000

Copy contour is_round information when copying polygons


---

diff --git a/src/polygon1.c b/src/polygon1.c
index 7c372c0..296841f 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -3518,6 +3518,10 @@ poly_CopyContour (PLINE ** dst, PLINE * src)
   (*dst)->xmin = src->xmin, (*dst)->xmax = src->xmax;
   (*dst)->ymin = src->ymin, (*dst)->ymax = src->ymax;
   (*dst)->area = src->area;
+  (*dst)->is_round = src->is_round;
+  (*dst)->cx = src->cx;
+  (*dst)->cy = src->cy;
+  (*dst)->radius = src->radius;
 
   for (cur = NEXT_EDGE (&src->head); cur != &src->head; cur = NEXT_VERTEX (cur))
     {
